Fabric.Identity.API/scripts/Install-IdPSS-Utilities.psm1

```diff
@@ -160,12 +160,15 @@ function Connect-AzureADTenant {
     param(
         [Parameter(Mandatory=$true)]
         [string] $tenantId,
-        [Parameter(Mandatory=$true)]
         [PSCredential] $credential
     )
 
     try {
-        Connect-AzureAD -Credential $credential -TenantId $tenantId | Out-Null
+        if($credential) {
```

Comment:
Remove the mandatory flag.  On the HC tenant, we use SSO, that redirect was causing the login to fail.  
If not credential is provided, then the AzureAD module handles that authentication portion, by opening up in a browser window which handles that case better.